### ISSUE TITLE
Localization do not work

### ISSUE THREAD

**Issue by PutinVladimir** [Ximi1970/systray-x#52]
Inside TB on plugin page I see translated plugin description string "Дополнение "Системный трей". Пожалуйста установите также приложение-компаньон" and thats all. Into plugin settings I see only templates like " __MSG_intro_line_1__"...

And Companion App I see in english.

**Comment by Ximi1970** [comment]
I cannot reproduce the MSG_intro problem but I updated the code yesterday a bit at that line. Make sure you got the latest updates. The english in the companion app was a problem with the language tag used by TB. For Russian it is just ru and not ru_RU as I used previously.  
There was also another problem with a dynamic option text change and the "ok" and "cancel" buttons translations in the app.
I downloaded the ru TB and checked. It should be solved now.

**Comment by PutinVladimir** [comment]
Some kind of mysticism. Now I turned on the computer and saw that the Russian language appeared in the plugin settings.
I tried running like
LANG=pt_BR thunderbird
LANG=de thunderbird
and saw only the plugin description string translated, and the settings in English(not templates).
I assume they are cached somewhere.

In any case I didn't see translation any language in companion.

About "ru" vs "ru_RU" checked few applications and it should be "ru" and it will work with any "ru_RU", "ru_UA" or "ru_KZ".

**Comment by PutinVladimir** [comment]
Rechecked develop branch - everything work fine.

**Comment by Ximi1970** [comment]
I do not know if Thunderbird looks at the LANG var. I use the `browser.i18n.getUILanguage()` to get the TB locale.  
Need to investigate this and maybe strengthen the locale handling somehow.  
Thnx for checking!

**Comment by fdellwing** [comment]
@Ximi1970 Can you look into this again? For some time now I only get _STRINGS_ instead of the fallback english strings. My TB is using german locale.

**Comment by Ximi1970** [comment]
Seems to work just fine for me..  
```bash  
LANG=de thunderbird
```
starts my default english version into a full german version.  
Keep in mind that you also need to install the Mozilla translations, in openSuSE the package is called ` MozillaThunderbird-translations-common`  
```bash
sudo zypper install MozillaThunderbird-translations-common   
```

**Comment by fdellwing** [comment]
My TB is perfectly fine in german, it's just this addons that does not work.

```
hi  thunderbird                                                 1:68.10.0+build1-0ubuntu0.18.04.1                   amd64        Email, RSS and newsgroup client with integrated spam filter
hi  thunderbird-locale-de                                       1:68.10.0+build1-0ubuntu0.18.04.1                   amd64        German language pack for Thunderbird
```

It would not be a problem for me, if the fallback would work. The `_STRINGS_` should never be shown, just show the english original.

**Comment by fdellwing** [comment]
It seems this is fixed with TB 68.12.0 from mozilla team.
